### ISSUE TITLE
[Suggestion] Hotbar accessibility

### ISSUE THREAD

**Issue by PopkaAkula** [tsunamayo/Starship-EVO#1820]
### Short Description of the suggestion:

As of now the hotbar in the game is white and looks like this:

![image](https://user-images.githubusercontent.com/37738021/80126204-0e9bce00-8593-11ea-8119-68fc56f2f45d.png)

While perfectly fine on any dark or contrasting background, it becomes almost impossible to see when you have something selected on the hotbar while the background is bright or a similar color as seen in this image:

![image](https://user-images.githubusercontent.com/37738021/80126399-53c00000-8593-11ea-9366-f53b7a53be63.png)

Perhaps the hotbar should be more opaque or should have a dark background on it in order to see what you have selected or else this could potentially be an accessibility issue.

**Comment by ZachZent** [comment]
![](https://cdn.discordapp.com/attachments/406664114249793536/702963590528893023/5fygtKo0YS7P7Vai49Cab_MBZAtgJQwxR7bga5k5bdg.png)

Something like this also works. Well same principle anyway

**Comment by tsunamayo** [comment]
yes very true.

**Comment by tsunamayo** [comment]
Better selection sprite available for 20w18c. Thanks
